cmd/neofs-cli/modules/object/util.go

```diff
@@ -278,8 +278,13 @@ func OpenSessionViaClient(ctx context.Context, cmd *cobra.Command, dst SessionPr
 	const sessionLifetime = 10 // in NeoFS epochs
 
 	common.PrintVerbose(cmd, "Opening remote session with the node...")
-
-	err := sessionCli.CreateSession(ctx, &tok, cli, *key, sessionLifetime)
+	endpoint, _ := cmd.Flags().GetString(commonflags.RPC)
+	currEpoch, err := internal.GetCurrentEpoch(ctx, endpoint)
+	common.ExitOnErr(cmd, "can't fetch current epoch: %w", err)
+	exp := currEpoch + sessionLifetime
+	tok.SetNbf(currEpoch)
```

Comment:
well, i see, some my suggestion conflicts with the reallity. i would not set _some parts_ of a token here and _some_ on the `CreateSession` side cause it bug-prone IMO

can we pass `currEpoch` as a param instead? or any another alternative that does not make a caller set `nbf` and `iat` outside?

Reply:
Well, we can (made it in the last commit), however, actually it seems that at the beginning (https://github.com/nspcc-dev/neofs-node/pull/2521#discussion_r1301985286) when we were passing `lifetime` to the `CreateSession` it was more optimal in numbers of calls of `internal.GetCurrentEpoch`